### PR TITLE
Update Marquez Stylesheets

### PR DESCRIPTION
**Pull request by phixMe**
### Description

This PR updates the overall stylesheets and modernizes the look at feel of the application. It removes some dead elements and moves away from some libraries that are no longer supported. This contains **no addition or removal** of features.

#### Assorted Changes
- Fix fonts and integrate general purpose text rendering coponent.
- New theme!
- New Search Bar
- Removal of percentage based styling practices

### CODE REVIEW

File: src/components/core/text/MqText.tsx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+import { WithStyles, createStyles, fade } from '@material-ui/core'
+import classNames from 'classnames'
+import withStyles from '@material-ui/core/styles/withStyles'
+
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      lineHeight: 1.5,
+      fontSize: '.875rem',
+      fontFamily: '\'Graphik\', sans-serif',
```
Comment (julienledem) [inline src/components/core/text/MqText.tsx]: Can we use this font here?

**Review by julienledem — COMMENTED** [review]
This looks good. But I think the fonts have to be free.

**Review by wslulciuc — APPROVED** [review]
New CSS is 🔥 Amazing work, @phixMe!

File: src/components/core/text/MqText.tsx (hunk 1 above)
Comment (phixMe) [reply src/components/core/text/MqText.tsx]: Thanks for pointing this out.
We are now back to `Source Code Pro` and `Karla` which are both free google fonts.

**Pull request merged**